### ISSUE TITLE
Simple node message identifiers are not parsed correctly

### ISSUE THREAD

**Issue by balazsracz** [openlcb/OpenLCB_Java#254]
These issues are in the JMRI monitor. It's unclear whether this is a problem in the parsing codebase of openlcb.jar or in the JMRI monitor code though.

- Node initialization with MTI 0x101 is displayed as an unknown message in the monitor.
- We should check whether the node id verified / simple subset is supported.

**Comment by dpharris** [comment]
Balazs et al --

Hmmm.  I looked at my MTIAlocations table, and it does not appear ...
should I add it?

But worse, if I change the Simple-protocol bit to 1, the MTI becomes
0x0110, not 0x0101.

David

**Comment by balazsracz** [comment]
Yes, these messages are in the message network standard, therefore they should be added to mtiallocations.ods/pdf.

**Comment by dpharris** [comment]
I am suggesting that the standard has an error.

David

**Comment by balazsracz** [comment]
Maybe I don't understand what you are suggesting, maybe you could elaborate
on why you think the standard has an error here?

**Comment by bobjacobsen** [comment]
Solved by PR #264
